### PR TITLE
Add missing components to report tool

### LINKED ISSUE #3570: Report tool staleness
Pods running newer images of kafka and zookeeper do not contain tls-sidecar containers anymore. The report tool still tries to get their logs.
Also we are missing MM2 & Cruise control checks.

### PR DESCRIPTION
**Pull request by fvaleri**
This should close #3570 adding missing components to report tool.

### CODE REVIEW

**Comment by fvaleri** [comment]
Output example:

```shell
deployments
    deployment.apps/my-cluster-cruise-control
    deployment.apps/my-cluster-entity-operator
    deployment.apps/my-cluster-kafka-exporter
    deployment.apps/my-connect-connect
statefulsets
    statefulset.apps/my-cluster-kafka
    statefulset.apps/my-cluster-zookeeper
replicasets
    replicaset.apps/my-cluster-cruise-control-85474c9ff9
    replicaset.apps/my-cluster-entity-operator-8bc54fdd7
    replicaset.apps/my-cluster-kafka-exporter-56b579f595
    replicaset.apps/my-connect-connect-574c867fc9
    replicaset.apps/my-connect-connect-767b8dcb68
configmaps
    configmap/my-cluster-cruise-control-config
    configmap/my-cluster-entity-topic-operator-config
    configmap/my-cluster-entity-user-operator-config
    configmap/my-cluster-kafka-config
    configmap/my-cluster-zookeeper-config
    configmap/my-connect-connect-config
secrets
    secret/my-cluster-clients-ca
    secret/my-cluster-clients-ca-cert
    secret/my-cluster-cluster-ca
    secret/my-cluster-cluster-ca-cert
    secret/my-cluster-cluster-operator-certs
    secret/my-cluster-cruise-control-api
    secret/my-cluster-cruise-control-certs
    secret/my-cluster-entity-operator-certs
    secret/my-cluster-kafka-brokers
    secret/my-cluster-kafka-exporter-certs
    secret/my-cluster-zookeeper-nodes
services
    service/my-cluster-cruise-control
    service/my-cluster-kafka-0
    service/my-cluster-kafka-1
    service/my-cluster-kafka-2
    service/my-cluster-kafka-bootstrap
    service/my-cluster-kafka-brokers
    service/my-cluster-kafka-external-bootstrap
    service/my-cluster-zookeeper-client
    service/my-cluster-zookeeper-nodes
    service/my-connect-connect-api
poddisruptionbudgets
    poddisruptionbudget.policy/my-cluster-kafka
    poddisruptionbudget.policy/my-cluster-zookeeper
    poddisruptionbudget.policy/my-connect-connect
roles
    role.rbac.authorization.k8s.io/my-cluster-entity-operator
rolebindings
    rolebinding.rbac.authorization.k8s.io/my-cluster-entity-topic-operator-role
    rolebinding.rbac.authorization.k8s.io/my-cluster-entity-user-operator-role
networkpolicies
    networkpolicy.networking.k8s.io/my-cluster-network-policy-cruise-control
    networkpolicy.networking.k8s.io/my-cluster-network-policy-kafka
    networkpolicy.networking.k8s.io/my-cluster-network-policy-zookeeper
    networkpolicy.networking.k8s.io/my-connect-connect
pods
    pod/my-cluster-cruise-control-85474c9ff9-mw686
    pod/my-cluster-entity-operator-8bc54fdd7-4tg4s
    pod/my-cluster-kafka-0
    pod/my-cluster-kafka-1
    pod/my-cluster-kafka-2
    pod/my-cluster-kafka-exporter-56b579f595-k7wzd
    pod/my-cluster-zookeeper-1
    pod/my-cluster-zookeeper-2
    pod/my-connect-connect-767b8dcb68-6snpn
persistentvolumeclaims
    persistentvolumeclaim/data-0-my-cluster-kafka-0
    persistentvolumeclaim/data-0-my-cluster-kafka-1
    persistentvolumeclaim/data-0-my-cluster-kafka-2
    persistentvolumeclaim/data-my-cluster-zookeeper-0
    persistentvolumeclaim/data-my-cluster-zookeeper-1
    persistentvolumeclaim/data-my-cluster-zookeeper-2
ingresses
routes
    route.route.openshift.io/my-cluster-kafka-0
    route.route.openshift.io/my-cluster-kafka-1
    route.route.openshift.io/my-cluster-kafka-2
    route.route.openshift.io/my-cluster-kafka-bootstrap
clusterroles
    clusterrole.rbac.authorization.k8s.io/strimzi-entity-operator
    clusterrole.rbac.authorization.k8s.io/strimzi-kafka-broker
    clusterrole.rbac.authorization.k8s.io/strimzi-kafka-client
clusterrolebindings
podlogs
    my-cluster-cruise-control-85474c9ff9-mw686
    my-cluster-entity-operator-8bc54fdd7-4tg4s
    my-cluster-kafka-0
    my-cluster-kafka-1
    my-cluster-kafka-2
    my-cluster-kafka-exporter-56b579f595-k7wzd
    my-cluster-zookeeper-1
    my-cluster-zookeeper-2
    my-connect-connect-767b8dcb68-6snpn
clusteroperator
draincleaner
    deployment.apps/strimzi-drain-cleaner
    pod/strimzi-drain-cleaner-579f6f7db6-gbbq5
customresources
    kafkabridges.kafka.strimzi.io
        my-bridge
    kafkaconnectors.kafka.strimzi.io
        artemis-sink
        mysql-source
    kafkaconnects.kafka.strimzi.io
        my-connect
    kafkas.kafka.strimzi.io
        my-cluster
    kafkatopics.kafka.strimzi.io
        connect-cluster-configs
        connect-cluster-offsets
        connect-cluster-status
        consumer-offsets---84e7a678d08f4bd226872e5cdd4eb527fadc1c6a
        my-mysql.testdb.customers
        my-topic
        strimzi-store-topic---effb8e3e057afce1ecf67c3f5d8e4e3ff177fc55
        strimzi-topic-operator-kstreams-topic-store-changelog---b75e702040b99be8a9263134de3507fc0cc4017b
        strimzi.cruisecontrol.metrics
        strimzi.cruisecontrol.modeltrainingsamples
        strimzi.cruisecontrol.partitionmetricsamples
customresourcedefinitions
    kafkabridges.kafka.strimzi.io
    kafkaconnectors.kafka.strimzi.io
    kafkaconnects.kafka.strimzi.io
    kafkaconnects2is.kafka.strimzi.io
    kafkamirrormaker2s.kafka.strimzi.io
    kafkamirrormakers.kafka.strimzi.io
    kafkarebalances.kafka.strimzi.io
    kafkas.kafka.strimzi.io
    kafkatopics.kafka.strimzi.io
    kafkausers.kafka.strimzi.io
events
Report file report-10-03-2022_18-17-41.zip created
```

File: tools/report.sh [hunk 1]
```diff
@@ -36,21 +36,40 @@ if [[ $OC_INSTALLED = false && $KUBECTL_INSTALLED = false ]]; then
   error "There is no kubectl or oc installed"
 fi
 
+# check kube connectivity
+$KUBE_CLIENT version --request-timeout=10s 1>/dev/null
```
Comment (fvaleri) [inline tools/report.sh:40]: This is to avoid hangs when there is no connectivity.

**Review by scholzj — APPROVED** [review]
Nice contribution, thanks.

**Review by ppatierno — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks for the PR @fvaleri

**Pull request merged**